modules/openapi-generator/src/main/java/org/openapitools/codegen/templating/HandlebarsEngineAdapter.java

```diff
@@ -83,14 +82,22 @@ public TemplateSource sourceAt(String location) {
         return tmpl.apply(context);
     }
 
-    public TemplateSource findTemplate(TemplatingGenerator generator, String templateFile) {
-        for (String file : getModifiedFileLocation(templateFile)) {
+    public TemplateSource findTemplate(TemplatingExecutor generator, String templateFile) {
+        String[] possibilities = getModifiedFileLocation(templateFile);
+        for (String file : possibilities) {
             try {
                 return new StringTemplateSource(file, generator.getFullTemplateContents(file));
             } catch (Exception ignored) {
             }
         }
-        throw new TemplateNotFoundException(templateFile);
+
+        // allow lookup of files without extension modification (such as .openapi-generator-ignore, README.md, etc)
+        try {
```

Comment:
This is required because filenames without extensions would become `.hbs` or `.handlebars` in the above loop. A file which isn't intended to be handled like `REAMDE.md` would become `README.hbs` or `README.handlebars` and would result in template not found without this as-is attempt.